client/my-sites/plan-features-2023-grid/style.scss

```diff
@@ -442,6 +442,9 @@ $mobile-card-max-width: 440px;
 
 				&.plan-features-2023-grid__header-billing-info {
 					padding-bottom: 32px;
+					@include plans-2023-break-small {
```

Comment:
Can you target this without the media query? We are going to be removing these breakpoints soon, so best if we don't encode more of them?

Reply:
This is competing for specificity with

https://github.com/Automattic/wp-calypso/blob/9b883b20741d3f9e997c942c96826ef79fac0c81/client/my-sites/plan-features-2023-grid/style.scss#L587-L595

What would be the best approach to fix this that would also assist with the refactor 🤔. What are we replacing the breakpoints with exactly ?

I could of course just double the selectors on this class to surpass specificity, if that is what you meant? :)

Reply:
If there's no easy way, then lets leave it with media query. It will get cut out along with the other. 

@aneeshd16 is introducing "grid-size" flags I think (small, medium, large) programmatically. but not sure where we are with that right now.

Reply:
Got it, will add the breakpoint for now since cleaning up will anyway remove both breakpoints leaving this issues fixed.
CC - @aneeshd16 regarding possible cleanup that would be required.